roles/kubernetes/control-plane/templates/kubeadm-config.v1beta3.yaml.j2

```diff
@@ -447,10 +447,13 @@ clusterDNS:
 {% for dns_address in kubelet_cluster_dns %}
 - {{ dns_address }}
 {% endfor %}
+{% if container_checkpoint_enabled is defined and container_checkpoint_enabled -%}
+{% set kubelet_feature_gates = kubelet_feature_gates + ["ContainerCheckpoint=true"] -%}
+{% endif -%}
 {% if kubelet_feature_gates or kube_feature_gates %}
 {% set feature_gates = ( kubelet_feature_gates | default(kube_feature_gates, true) ) %}
 featureGates:
-{%   for feature in feature_gates %}
+{%   for feature in (feature_gates | unique) %}
```

Comment:
What's the reason for `unique` there ? (Maybe explicit that in the commit message ?)

Reply:
Just to avoid duplication, e.g. users accidentally copy more

Reply:
But that's not related to container checkpoint right ? Maybe put that in a separate PR ?

Reply:
I agree. I removed them from this PR.